src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1200,16 +1200,20 @@ public function resolveServices($value)
      *     }
      * }
      *
-     * @param string $name The tag name
+     * @param string $name
+     * @param bool   $throwOnAbstract
      *
      * @return array An array of tags with the tagged service as key, holding a list of attribute arrays
      */
-    public function findTaggedServiceIds($name)
+    public function findTaggedServiceIds($name, $throwOnAbstract = false)
```

Comment:
what about set `null` for now (and consider it as `false`), trigger deprecation warning if no argument value provided (`null`), change it to `true` by default for 4.0.